src/components/Person/Person.jsx

```diff
@@ -1 +1,28 @@
-// export const Person = ({ person }) => ();
+import './Person.scss';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${name}`}
+      </h2>
+      {Boolean(age) && (
+        <p className="Person__age">
+          {`I am ${age}`}
+        </p>
+      )}
+      <p className="Person__partner">
+        {isMarried ? `${partnerName} is my ${sex === 'm' ? 'wife' : 'husband'}`
+          : `I am not married`}
```

Comment:
Don't nest ternary. Create variable for `${sex === 'm' ? 'wife' : 'husband'}`

Reply:
Also, pay attention to formatting

<img width="1051" alt="image" src="https://github.com/mate-academy/react_person/assets/86789161/f773bcf9-8d59-4b1b-88b5-72d45b3efe6a">